google-cloud-spanner/lib/google/cloud/spanner/errors.rb

```diff
@@ -20,6 +20,16 @@ module Cloud
     module Spanner
       ##
       # @private
+      # # CommitError
+      #
+      # Signal that the Transaction has been committed.
```

Comment:
why do we want to allow user to explicitly call commit, if we are automatically doing that at the end of block.